frontend/src/scenes/surveys/SurveyCustomization.tsx

```diff
@@ -134,6 +135,43 @@ export function Customization({ appearance, surveyQuestionItem, onAppearanceChan
                         checked={appearance?.shuffleQuestions}
                     />
                 </div>
+                <div className="mt-1">
```

Comment:
Up to you, but we might want to gate by a feature flag here.

Reply:
FYI, I'm electing to not feature flag this, because we 
1. don't have that many users on surveys, so a staged rollout here seems like overkill
2. this is really easy to revert and doesn't leave many lasting (and scary) changes to our schema, etc, if we do.